ui/components/src/utils/axis.ts

```diff
@@ -12,15 +12,15 @@
 // limitations under the License.
 
 import merge from 'lodash/merge';
-import type { YAXisComponentOption } from 'echarts';
+import type { XAXisComponentOption, YAXisComponentOption } from 'echarts';
 import { formatValue, UnitOptions } from '@perses-dev/core';
 
 /*
- * Populate yAxis properties, returns an Array since multiple y axes will be supported in the future
+ * Populate yAxis or xAxis properties, returns an Array since multiple axes will be supported in the future
  */
-export function getYAxes(yAxis?: YAXisComponentOption, unit?: UnitOptions) {
+export function formatAxesValues(axis?: YAXisComponentOption | XAXisComponentOption, unit?: UnitOptions) {
```

Comment:
Could make sense to call this `getAxisWithFormatting` / `getFormattedAxis` or something since I think it still returns the entire axis object

Reply:
(Especially since many of the call sites are naming the output of this function `axis`, so it could be like `yAxis: getFormattedAxis(...)`)

Reply:
I like that, updated it to `getFormattedAxis`!